saleor/graphql/product/schema.py

```diff
@@ -296,8 +302,12 @@ def resolve_product_type(self, info, id):
     def resolve_product_types(self, info, **kwargs):
         return resolve_product_types(info, **kwargs)
 
-    def resolve_product_variant(self, info, id):
-        return graphene.Node.get_node_from_global_id(info, id, ProductVariant)
+    def resolve_product_variant(self, info, id=None, sku=None):
+        validate_one_of_args_is_in_query("id", id, "sku", sku)
+        if id:
+            return graphene.Node.get_node_from_global_id(info, id, ProductVariant)
+        if sku:
```

Comment:
I assume that `validate_one_of_args_is_in_query` already checked if skur or id is here.  So second if is not needed 
```
validate_one_of_args_is_in_query("id", id, "sku", sku)
if id:
    return graphene.Node.get_node_from_global_id(info, id, ProductVariant)
 return resolve_product_varaint_by_sku(info, sku=sku)
```